regulations/tasks.py

```diff
@@ -24,55 +25,70 @@
 
 from django.conf import settings
 from django.template import loader
+from django.utils.crypto import get_random_string
 
 from regulations.models import FailedCommentSubmission
 logger = get_task_logger(__name__)
 
 
 @shared_task(bind=True)
-def submit_comment(self, body):
+def submit_comment(self, comments, form, meta):
```

Comment:
Would `additional_comment_fields` be a better name than `form`?
Would `signed_url` be a better name than `meta`?